### ISSUE TITLE
readme backend uses unclear

### ISSUE THREAD

**Issue by wighawag** [parcel-bundler/watcher#136]
Hi, in the doc there is both these sentences :

> @parcel/watcher has the following watcher backends, listed in priority order:
> 
> [FSEvents](https://developer.apple.com/documentation/coreservices/file_system_events) on macOS
> [Watchman](https://facebook.github.io/watchman/) if installed
> [inotify](http://man7.org/linux/man-pages/man7/inotify.7.html) on Linux
> [ReadDirectoryChangesW](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365465%28v%3Dvs.85%29.aspx) on Windows

and 

> @parcel/watcher has the following watcher backends, listed in priority order:
> 
> [FSEvents](https://developer.apple.com/documentation/coreservices/file_system_events) on macOS
> [Watchman](https://facebook.github.io/watchman/) if installed
> [fts](http://man7.org/linux/man-pages/man3/fts.3.html) (brute force) on Linux
> [FindFirstFile](https://docs.microsoft.com/en-us/windows/desktop/api/fileapi/nf-fileapi-findfirstfilea) (brute force) on Windows

The are listed in 2 different sections, one for watching, one for querying but the wording make me wonder if I should worry about this if I intent to use the watcher to watch file like I do with chokidar

Not really interested in the watchman option as I do not expect all the user of my tool to want to install it
so my worry is on Linux.

Basically should I use chokidar or @parcel/watcher ?

**Comment by mischnic** [comment]
For watching (so to listen for file changes while your tool is running), macOS, Linux and Windows all have some native backend that works fine. Even without watchman.

But for querying (so to answer "which files changed since you last ran you tool"), there's only a native backend on macOS and all other platforms use bruteforce or Watchman if available.

So you should be fine as far as I understand you
